samples/model-builder/experiment_tracking/log_classification_metrics_sample.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import List, Optional
+
+from google.cloud import aiplatform
+
+
+#  [START aiplatform_sdk_log_classification_metrics_sample]
+def log_classification_metrics_sample(
+    experiment_name: str,
+    run_name: str,
+    project: str,
+    location: str,
+    labels: Optional[List[str]] = None,
+    matrix: Optional[List[List[int]]] = None,
+    fpr: Optional[List[float]] = None,
+    tpr: Optional[List[float]] = None,
+    threshold: Optional[List[float]] = None,
+    display_name: Optional[str] = None,
```

Comment:
It seems like some/all values are used for the most important part of this sample, `log_classification_metrics` method, but they're marked as all Optional. I'd guess that this is because of the underlying API, but looks a bit confusing to me at first glance. If they really are all optional then feel free to disregard this comment.

Reply:
Yes, this is because of the underlying API. This `log_classification_metrics` allows users to log both confusion matrices and ROC curves. `labels` and `matrix` are for confusion matrices and `fpr`, `tpr`, and `threshold` are for ROC curves. If the user only want to log a confusion matrix but not a ROC curve, then `fpr`, `tpr`, and `threshold` should not be set.